homework/Vladislav Urbanski/Home work 6/Credit Card/src/com/vladis1350/bean/CreditCard.java

```diff
@@ -0,0 +1,110 @@
+package com.vladis1350.bean;
+
+import java.sql.SQLOutput;
+
+public class CreditCard {
+    private long numberCard;
+    private int pinCode;
+    private double balance;
+    private double creditLimit;
+    private double loanDebt;
+
+    public CreditCard() {
+    }
+
+    public CreditCard(long numberCard, int pinCode) {
+        this.numberCard = numberCard;
+        this.pinCode = pinCode;
+        this.balance = 0.0;
+        this.loanDebt = 0.0;
+    }
+
+    public boolean deposit(int pinCode, double sumDeposit) {
+        if (this.pinCode != pinCode) {
+            System.out.println("Invalid pin code!");
+            return false;
+        } else if (this.loanDebt <= sumDeposit) {
+            double balanceAmount = sumDeposit - this.loanDebt;
+            this.loanDebt = 0;
+            this.balance += balanceAmount;
+        } else if (this.loanDebt >= sumDeposit) {
+            this.loanDebt -= sumDeposit;
+        } else {
+            this.balance += sumDeposit;
+        }
+        return true;
+    }
+
+    public boolean withdraw(int pinCode, double sumWithdraw) throws Exception {

```

Comment:
withdraw and deposit should be like entry point to other methods.
see what I wrote to vov4eg